test/cuda_sims.jl

```diff
@@ -148,7 +148,7 @@ end
     F = t -> t ≥ 1.1 ? F₂ : F₁)
 
   prob = ODEProblem(cpu_fₘ, cpu_u₀, (0, tₑ), cpu_constants_and_parameters)
-  cpu_soln = solve(prob, Tsit5())
+  cpu_soln = solve(prob, Tsit5(), save_everystep=false, save_idxs=[:U])
```

Comment:
Don't need this `save_idxs` since `:U` is the only thing we can track.

Reply:
Can't :V also be tracked?

Reply:
Yeah you're right, I thought this was for Heat.